exercises/D-predicates/exercise.js

```diff
@@ -6,10 +6,16 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative( number ){
+   return typeof number === "number";
+}
+  isNegative(5)
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10( number ){
+    return typeof number === "number";
+}
+   isBetweenZeroAnd10(10)
```

Comment:
These two function are just checking the type of the parameter being passed in to the function rather than following the functionality inferred by the function name.